examples/dropwizard/pom.xml

```diff
@@ -0,0 +1,235 @@
+
+        <jib.to.image>gcr.io/REPLACE-WITH-YOUR-GCP-PROJECT/image-built-with-jib</jib.to.image>
+        <jib-maven-plugin.version>0.10.0</jib-maven-plugin.version>
+        <jib.extraContext>src/main/jib</jib.extraContext>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>io.dropwizard</groupId>
+                <artifactId>dropwizard-bom</artifactId>
+                <version>${dropwizard.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.dropwizard</groupId>
+            <artifactId>dropwizard-core</artifactId>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+
+            <!-- Used to copy Dropwizard's YAML configuration into a directory Jib can locate it -->
+            <plugin>
+                <groupId>com.coderplus.maven.plugins</groupId>
+                <artifactId>copy-rename-maven-plugin</artifactId>
+                <version>1.0</version>
+                <executions>
+                    <execution>
+                        <id>copy-file</id>
+                        <phase>generate-sources</phase>
+                        <goals>
+                            <goal>copy</goal>
+                        </goals>
+                        <configuration>
+                            <sourceFile>${dropwizard.server.config}</sourceFile>
+                            <destinationFile>${jib.extraContext}/${dropwizard.server.config}</destinationFile>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+
+            <!-- NOTE: Workaround for cleaning populated src/main/jib contents since jib.extraDirectory is incubating -->
+            <plugin>
+                <artifactId>maven-clean-plugin</artifactId>
+                <version>3.1.0</version>
+                <configuration>
+                    <filesets>
+                        <fileset>
+                            <directory>src/main/jib</directory>
+                            <includes>
+                                <include>${dropwizard.server.config}</include>
+                            </includes>
+                        </fileset>
+                    </filesets>
+                </configuration>
+            </plugin>
+
+            <plugin>
+                <groupId>com.google.cloud.tools</groupId>
+                <artifactId>jib-maven-plugin</artifactId>
+                <version>${jib-maven-plugin.version}</version>
+                <configuration>
+                    <to>
+                        <image>${jib.to.image}</image>
+                        <tags>
+                            <tag>${project.version}</tag>
+                            <tag>latest</tag>
+                        </tags>
```

Comment:
If `latest` is not required, I think we can just git rid of the entire `<tags>` part to further simplify it.